frontends/precinct-scanner/src/utils/save_cvr_export_to_usb.ts

```diff
@@ -0,0 +1,68 @@
+import { ElectionDefinition } from '@votingworks/types';
+import {
+  generateFilenameForScanningResults,
+  SCANNER_RESULTS_FOLDER,
+  usbstick,
+  generateElectionBasedSubfolderName,
+} from '@votingworks/utils';
+import path from 'path';
+import fileDownload from 'js-file-download';
+import * as scan from '../api/scan';
+import { MachineConfig } from '../config/types';
+
+/**
+ * Saves the current results from the scanning service to the usb drive, if present.
+ * @param openFilePickerDialog Toggles whether to open the file picker dialog and allow the user to customize the filename / location to save.
+ */
+export async function saveCvrExportToUsb(
+  electionDefinition: ElectionDefinition,
+  machineConfig: MachineConfig,
+  scannedBallotCount: number,
+  isTestMode: boolean,
+  openFilePickerDialog: boolean
```

Comment:
It's hard to understand what all these arguments are when they're just positional. Consider making this an object with options.